src/main/kotlin/com/codingnomads/mentor_portal_api/controllers/MentorController.kt

```diff
@@ -0,0 +1,26 @@
+package com.codingnomads.mentor_portal_api.controllers
+
+import com.codingnomads.mentor_portal_api.entities.business.Mentor
+import com.codingnomads.mentor_portal_api.handlers.MentorHandler
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.bind.annotation.*
+
+@CrossOrigin(origins = ["http://localhost:5000"])

```

Comment:
Please investigate how to handle this at the app level, rather than duplicating the code on every controller.